productivity-suite/piercing-library/src/piercing-fragment-outlet.ts

```diff
@@ -80,7 +80,28 @@ export class PiercingFragmentOutlet extends HTMLElement {
 
   disconnectedCallback() {
     if (this.fragmentHost) {
-      unmountedFragmentIds.add(this.fragmentHost.fragmentId);
+      const fragmentId = this.fragmentHost.fragmentId;
```

Comment:
Can you add a comment before this block explaining what we are doing here? Thanks